charts/internal/control-plane/values.yaml

```diff
@@ -41,6 +42,17 @@ authnWebhook:
     issuerUrl: https://tokenissuer/dex
     issuerClientId: myClientId
 
+splunkAuditWebhook:
+  enabled: false
+  port: 443
+  acceptInsecureCert: false
+  splunkAuditWebhookUrl: https://splunk-audit-webhook.shoot--local--myshootname.svc.cluster.local/audit
```

Comment:
This variable name is a little redundant because it's now beneath `splunkAuditWebhook` anyway. Just `url` should be enough. Same for `splunkHECEndpoint` -> `HECEndpoint`.